server/internal/server/milvus.go

```diff
@@ -0,0 +1,98 @@
+package server
```

Comment:
How about moving this to a separate package like `milvus`? We can then change the struct name to `Store` or `Client`.